src/patternfly/components/Sidebar/sidebar.scss

```diff
@@ -88,6 +88,16 @@ $pf-c-sidebar__panel--list--width: ("default", 25, 33, 50, 66, 75, 100);
     }
   }
 
+  &.pf-m-full-height {
+    --pf-c-sidebar__main--AlignItems: stretch;
+
+    height: 100%;
+
+    .pf-c-sidebar__main {
```

Comment:
just a nit, you could also remove the `height: 100%` above and change this selector to this. maybe looks cleaner? could just be me, so up to you :)

```
&,
.pf-c-sidebar__main {
  height: 100%;
}
```